build/image_build.go

```diff
@@ -95,6 +96,11 @@ func BuildImage(imageName, rootDir string) error {
 	}
 	defer imageBuildResponse.Body.Close()
 
+	_, err = io.Copy(ioutil.Discard, imageBuildResponse.Body)
```

Comment:
I added this back because the image wasn't building without it. 

Reference: https://stackoverflow.com/questions/64510928/disable-verbose-stdout-in-docker-golang-sdk-for-client-imagepull

Reply:
The output is still suppressed like before.

Reply:
Can you try this?

```
if _, err := io.Copy(ioutil.Discard, imageBuildResponse.Body); err != nil {
       return err
}
```

Reply:
done